File: src/model/poe-api/StashTab.ts
import { Item } from "./Item";

export enum StashTabType {
  PremiumStash = "PremiumStash",
}

export interface StashTab {
  accountName: string;
  lastCharacterName: string;
  id: string;
  stash: string;
  stashType: StashTabType;
  items: Array<Item>;
  public: boolean;
  league: string;
}

File: src/go/model/poeapimodel/StashTab.go
package poeapimodel

// StashTab is exported
type StashTab struct {
	AccountName       string `json:"accountName"`
	LastCharacterName string `json:"lastCharacterName"`
	ID                string `json:"id"`
	Stash             string `json:"stash"`
	StashType         string `json:"stashType"`
	Items             []Item `json:"items"`
	Public            bool   `json:"public"`
	League            string `json:"league"`
}
